reclass/datatypes/parameters.py

```diff
@@ -119,7 +120,25 @@ def _extend_list(self, cur, new, path):
             ret.append(self._merge_recurse(None, new[i], path.new_subpath(offset + i)))
         return ret
 
-    def _merge_dict(self, cur, new, path):
+    def _merge_dict(self, cur, new, path, initmerge):
+        """Merge a dictionary with another dictionary.
+
+        Iterate over keys in new. If this is not an initialization merge and
+        the key begins with PARAMETER_DICT_KEY_OVERRIDE_PREFIX, override the
+        value of the key in cur. Otherwise deeply merge the contents of the key
+        in cur with the contents of the key in _merge_recurse over the item.
+
+        Args:
+            cur (dict): Current dictionary
+            new (dict): Dictionary to be merged
+            path (string): Merging path from recursion
+            initmerge (bool): True if called as part of entity init
```

Comment:
What is an example of a case in which a `Parameters` object performs a merge as part of `Entity` initialization? I don't see any merges that occur in [`Entity.__init__`](https://github.com/Rtzq0/reclass/blob/a247e94930f5cb88aa2c4abac0b9e609e88a58a2/reclass/datatypes/entity.py#L19-L36)